File: lib/plugin_constants.dart
class PluginConstants {
  static const String _baseEventChannelName = 'internal_hifi_plugin_baseEventChannel';
  String posTrackEventChannel = '$_baseEventChannelName/posTrackEventChannel';
  String playStateEventChannel = '$_baseEventChannelName/playStateEventChannel';
  String deviceStateEventChannel = '$_baseEventChannelName/deviceStateEventChannel';
  String errorEventChannel = '$_baseEventChannelName/errorEventChannel';
  String metadataEventChannel = '$_baseEventChannelName/metadataEventChannel';
  // Player state constants
  int STATE_IDLE = 1;
  int STATE_BUFFERING = 2;
  int STATE_READY = 3;
  int STATE_ENDED = 4;
}

File: android/src/main/kotlin/com/hifi/turntableplugin/internal_hifi_plugin/PluginConstants.kt
package com.hifi.turntableplugin.internal_hifi_plugin

object PluginConstants {
    private var baseEventChannelName: String = "internal_hifi_plugin_baseEventChannel"
    var posTrackEventChannel : String = this.baseEventChannelName + "/posTrackEventChannel"
    var playStateEventChannel : String = this.baseEventChannelName + "/playStateEventChannel"
    var deviceStateEventChannel : String = this.baseEventChannelName + "/deviceStateEventChannel"
    var metadataEventChannel : String = this.baseEventChannelName + "/metadataEventChannel"
    var errorEventChannel : String = this.baseEventChannelName + "/errorEventChannel"
}